python/console/console.py

```diff
@@ -59,7 +59,7 @@ def show_console():
     # Shows help on first launch of the console
     settings = QgsSettings()
     if settings.value('pythonConsole/contextHelpOnFirstLaunch', True, type=bool):
-        QgsContextHelp.run("PythonConsole")
+        QgsHelp.openHelp("../pyqgis_developer_cookbook/intro.html#python-console")
```

Comment:
There's also http://docs.qgis.org/2.18/en/docs/user_manual/plugins/python_console.html more detailed (which we could improve with link to PyQGIS Doc) and that I'd rather use